src/pages/error.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const NotMatch = () => (
+  <div className="no-match">
+    <h3>Whoops! Page not found!</h3>
+  </div>
+);
+
+export default NotMatch;
```

Comment:
- [ ] Nice job rendering a page when the wrong `URL` endpoint is inputted. Kudos 👏